imap/index.c

```diff
@@ -4229,7 +4237,7 @@ static void index_fetchflags(struct index_state *state,
     bit32 flagmask = 0;
     struct index_map *im = &state->map[msgno-1];
 
-    prot_printf(state->out, "* %u FETCH (FLAGS ", msgno);
+    prot_puts(state->out, "FLAGS ");
```

Comment:
The comment above the function still says "Also sends preceeding * FLAGS if necessary", but that's not quite right anymore.

It also says "Does not send the terminating close paren or CRLF", and it's true that it doesn't send the closing paren or CRLF for the whole `FETCH (...)`, but it does send the closing paren for the `FLAGS (...)` clause, so the second-last line of the function is it sending a `)` -- which makes the comment look wrong at first glance, and it takes some careful reading to figure out what's really going on.

Reply:
Fixed the comment